gh-pr-context.py

```
@@ -1,22 +1,494 @@
+    result = subprocess.run(args, capture_output=True, text=True)
+    if result.returncode != 0:
+        return None, False
+    return result.stdout.strip().replace("\r", ""), True
+
+
+def check_deps():
+    from shutil import which
+    for cmd in ("gh", "git"):
+        if not which(cmd):
+            die(f"{cmd} is required but not found on PATH")
+    out, rc = run_cmd(["git", "rev-parse", "--git-dir"])
```

Ruff (0.15.12)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)